frontend/apps/docs/components/Banner/JackAnimationForDark.tsx

```diff
@@ -19,7 +19,7 @@ const RiveComponent = () => {
 
 export const JackAnimationForDark: FC = () => {
   return (
-    <div className="w-[1.875rem] h-[1.875rem]">
+    <div className="w-[20px] h-[20px] sm:w-[1.875rem] sm:h-[1.875rem]">
```

Comment:
[nits]
I thought it would be better to unify the unit of size with `rem`.

```suggestion
    <div className="w-[1.25rem] h-[1.25rem] sm:w-[1.875rem] sm:h-[1.875rem]">
```